pkgs/tools/networking/xdp-tools/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchFromGitHub, libbpf, elfutils, libelf, zlib, libpcap
+, clang, llvm, gnumake, gcc, pkgconfig, m4, emacs-nox, wireshark-cli
+}:
+
+let
+  version = "1.2.5";
+  projectSha256 = "sha256-Kyay5j+87nOZ9C+DI8MI6zhkWqspIHiTfW9Di5uVWzY=";
+
+  src = fetchFromGitHub {
+    owner = "xdp-project";
+    repo = "xdp-tools";
+    rev = "v${version}";
+    sha256 = projectSha256;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "xdp-tools";
+  inherit version src;
+
+  nativeBuildInputs = [
+    libbpf
+    elfutils
+    libelf
+    zlib
+    libpcap
```

Comment:
`libbpf` `libelf` `libpcap` and some others should not belong to `nativeBuildInputs`. Put them into `BuildInputs`.